docs/api/index.md

```diff
@@ -191,8 +191,7 @@ appear in a request, a `422 Unprocessable Entity` will be returned.
 |`offset`|int|0|Where to start the listing(the offset of the first result)|
 
 Please note that there's a limit on the maximum offset or page you can require.
```

Comment:
"...maximum offset or page you can..." -> "...maximum offset or page number you can..."